feature/manager.go

```diff
@@ -91,6 +91,21 @@ func newManager(cfg Config, desc setDesc) (*Manager, error) {
 		if cfg.NoAnchors {
 			raw.Unset(lnwire.AnchorsZeroFeeHtlcTxOptional)
 			raw.Unset(lnwire.AnchorsZeroFeeHtlcTxRequired)
+
+			// If anchors are disabled, then we also need to
```

Comment:
Hmm, seems we could convert all other instances above to use logic like this as well, particularly the `notlvonion` part of it.

Reply:
+1, we can add a function with the logic of `disable feature X and features that depend on X (recursively!)`

Reply:
+1